imports/ui/components/reviews/review_page.js

```diff
@@ -21,8 +21,7 @@ Template.reviewPage.onCreated(function() {
 
 Template.reviewPage.helpers({
   review: () => {
-    var slug = FlowRouter.getParam('slug');
-    return Reviews.findOne({ slug: slug });
+    return Reviews.findOne({});
```

Comment:
This could cause a bug if, for example, we have other reviews in scope (such as reviews in a sidebar).